### PR TITLE
update handbook project icons : they are now responsive

### LINKED ISSUE #2290: [Handbook] Icons not mobile responsive
**Description**
Icons on the [Projects](https://layer5.io/community/handbook/projects) page are not mobile responsive.

**Expected Behavior**
Make all the icons bigger in mobile view and correctly aligned to the text. Also, change the `5` icon to one without the dark background.

**Screenshots**
projects

**Environment:**
- Host OS: 
- Browser: 

---

[Optional] **Additional Context**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by shubham-tyagi**
Signed-off-by: shubham-tyagi <raj.tyagi2000@gmail.com>

**Description**
I was going to apply media queries for responsiveness but the text size remains similar in almost all devices so I just applied the same size for all icons in all devices.

This PR fixes #2290

**Notes for Reviewers**
Please point out if I missed something or if something else is needed to be done.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Community/Handbook/Handbook.style.js [hunk 1]
```diff
@@ -168,6 +174,18 @@ export const HandbookWrapper = styled.div`
       background-color: ${(props) => props.theme.secondaryLightColor};
       border-color: ${(props) => props.theme.secondaryLightColor};
     }
+    
+    .p-icon{
```
[the remark this answers is not in the document]
Comment (shubham-tyagi) [reply src/sections/Community/Handbook/Handbook.style.js:178]: does para-icon and head-icon works ? @warunicorn19

**Comment by shubham-tyagi** [comment]
Hey, I have made the required changes. Please point out if I missed something or if anything else is needed to be done.

**Comment by l5io** [comment]
🚀 Preview for commit c013ba9fec94edec2790c427759468836248d4c1 at: https://617db18b8528ac25b395d38f--layer5ng.netlify.app

**Review by Anita-ihuman — APPROVED** [review]
The buttons are responsive now. @shubham-tyagi great job.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@shubham-tyagi everything looks good, just one minor change, could you please keep all the headings and the content below that equidistant for all the projects, for example, the space between "Meshery" and the content below is fine, but the space between "Service Mesh Performance" and content below is a little less, hope I made sense?

**Comment by shubham-tyagi** [comment]
@warunicorn19 yeah that does makes sense. I will make that change. Can you confirm the size of the project title icons for laptops? Are they a bit big or are they fine?

**Comment by warunicorn19** [comment]
It's a bit bigger than before I think, but I think it looks nice. @debo19 views?

**Comment by debo19** [comment]
I think it's fine, looks good

**Comment by l5io** [comment]
🚀 Preview for commit 686b2d6083b11a178803679d5324675c5d07081d at: https://617fbfcf97042d9c25c0b6c9--layer5ng.netlify.app

File: src/sections/Community/Handbook/projects.js [hunk 2]
```diff
@@ -25,99 +25,88 @@ const Maintainer = () => {
...
               </h3>
             </p>
             <p>
-              Its service mesh community represents the largest collection of
-              service Mesh projects.Emerging projects like GetNighthawk{" "}
-              <img width="2.5%" src={hawkIcon} /> , community with{" "}
-              <img width="2.5%" src={meshmateicon} /> MeshMates, catch-all org,
-              “home base”. <br />
+							Its service mesh community represents the largest collection of
+							service Mesh projects.Emerging projects like GetNighthawk{" "}
```
Comment (warunicorn19) [inline src/sections/Community/Handbook/projects.js:35]:
```suggestion
							service Mesh projects.Emerging projects like Nighthawk{" "}
```

File: src/sections/Community/Handbook/projects.js [hunk 3]
```diff
@@ -25,99 +25,88 @@ const Maintainer = () => {
...
+              <img className="project-description-icon" src={hawkIcon} /> , community with{" "}
+              <img className="project-description-icon" src={meshmateicon} /> MeshMates, catch-all
```
Comment (warunicorn19) [inline src/sections/Community/Handbook/projects.js:37]: add a link to the meshmates page here.

File: src/sections/Community/Handbook/projects.js [hunk 4]
```diff
@@ -25,99 +25,88 @@ const Maintainer = () => {
...
+									written in Rust. These modules can be used to implement
+									multi-tenancy or to implement per-user rate-limiting in your
+									application's endpoints. This application was first
+									demonstrated at DockerCon 2020. <br />
                 </p>
                 <li>
                   <h4>
                     <a href="https://layer5.io/projects/getnighthawk">
-                      GetNightHawk :
+											GetNightHawk :
```
Comment (warunicorn19) [inline src/sections/Community/Handbook/projects.js:69]:
```suggestion
											Nighthawk :
```

File: src/sections/Community/Handbook/projects.js [hunk 5]
```diff
@@ -25,99 +25,88 @@ const Maintainer = () => {
...
+									Image Hub is a sample application written to run on Consul for
+									exploring WebAssembly modules used as Envoy filters are
+									written in Rust. These modules can be used to implement
+									multi-tenancy or to implement per-user rate-limiting in your
+									application's endpoints. This application was first
+									demonstrated at DockerCon 2020. <br />
                 </p>
                 <li>
                   <h4>
                     <a href="https://layer5.io/projects/getnighthawk">
```
Comment (warunicorn19) [inline src/sections/Community/Handbook/projects.js:68]:
```suggestion
                    <a href="https://layer5.io/projects/nighthawk">
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
I'm sorry to keep doing this @shubham-tyagi 😅, just tiny changes

**Comment by warunicorn19** [comment]
Also head's up: merge conflicts

File: src/sections/Community/Handbook/projects.js (hunk 3 above)
Comment (shubham-tyagi) [reply src/sections/Community/Handbook/projects.js:37]: > I'm sorry to keep doing this @shubham-tyagi sweat_smile, just tiny changes

I don't mind it at all. Every time there is something to learn whenever I see changes required in a pull request, haha. You all have been very patient with my pull requests.

**Comment by l5io** [comment]
🚀 Preview for commit 77884881ca1fee3bd30aed15a196aa2302ac9ca1 at: https://618294054c5a2322556ecc06--layer5ng.netlify.app

File: src/sections/Community/Handbook/projects.js [hunk 6]
```diff
@@ -25,99 +25,88 @@ const Maintainer = () => {
...
             <p>
-              Its service mesh community represents the largest collection of
-              service Mesh projects.Emerging projects like GetNighthawk{" "}
-              <img width="2.5%" src={hawkIcon} /> , community with{" "}
-              <img width="2.5%" src={meshmateicon} /> MeshMates, catch-all org,
-              “home base”. <br />
+							Its service mesh community represents the largest collection of
+							service Mesh projects.Emerging projects like Nighthawk{" "}
+              <img className="project-description-icon" src={hawkIcon} /> , community with{" "}
+              <a href="https://layer5.io/community/meshmates"><img className="project-description-icon" src={meshmateicon} /> MeshMates </a>, catch-all org, “home base”.
```
Comment (warunicorn19) [inline src/sections/Community/Handbook/projects.js:37]:
```suggestion
              <Link to="/community/meshmates"><img className="project-description-icon" src={meshmateicon} /> MeshMates </Link>, catch-all org, “home base”. 
```

File: src/sections/Community/Handbook/Handbook.style.js [hunk 7]
```diff
@@ -1,28 +1,32 @@
 import styled from "styled-components";
```
Comment (warunicorn19) [inline src/sections/Community/Handbook/Handbook.style.js:1]:
```suggestion
import styled from "styled-components";
import { Link } from "gatsby";
```

**Comment by l5io** [comment]
🚀 Preview for commit 8aa3b6896bf49963ce9594de280eff23ed358096 at: https://6182a7b9af5e2307edf565b3--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM, Thanks @shubham-tyagi

**Pull request merged**